cmd/migrate_intakes/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
```

Comment:
Gotta research this. Is it like Commander in Node?

Reply:
Yeah, it's a tool for helping build CLI commands in Go!